cluster/saltbase/salt/generate-cert/make-ca-cert.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [ "${DEBUG}" == "true" ]; then
+if [ "${DEBUG:-}" == "true" ]; then
```

Comment:
It's just that now the whole script fails because of that line, and says something like `unset variable DEBUG. Exiting...`.
This fixes so it debugs if the flag is set, otherwise not